api/app/geotiff_from_stac_api.py

```diff
@@ -114,7 +124,11 @@ def get_geotiff(
     """Generate a geotiff and return presigned download url"""
     s3_filename = generate_geotiff_and_upload_to_s3(collection, bbox, date, band)
 
-    s3_client = boto3.client("s3")
+    s3_client = boto3.client(
+        "s3",
+        aws_access_key_id=STAC_AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=STAC_AWS_SECRET_ACCESS_KEY,
+    )
```

Comment:
I needed to add these for local development. Considering that this shouldn't have negative impacts and eases development, I thought I'd leave it in

Reply:
The default AWS account is the one in which the PRISM instance sits, which is (can be) different from the STAC one.

I think we should avoid linking the two if possible. Locally you can use the default boto3 env variables and set them up like this - https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#environment-variables
Or use the `aws configure` command

If this does not work we can figure out a better way to keep the credentials separate.

Reply:
Oh interesting, I didn't realize that. Thanks @ericboucher. I've reverted this change for now and will experiment more with this once I have full access to AWS to test it out